randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1052,7 +1100,12 @@ Extra - asset_id: 105
   > Door to Alpha Power Bomb Room
       Trivial
   > Door to Red Tower Save Station
-      Hi-Jump Boots or Space Jump
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          # IBJ
+          Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+          # Free the Catepillar below and Ice Climb up
```

Comment:
This should either be combined with movement, or with a seperate ice beam abuse trick. im leaning towards the latter. same for the other bug one.

Reply:
Added Ice Beam Abuse